opencti-platform/opencti-front/src/private/components/entities/Sectors.jsx

```diff
@@ -13,11 +13,16 @@ import { KNOWLEDGE_KNUPDATE } from '../../../utils/hooks/useGranted';
 import withRouter from '../../../utils/compat-router/withRouter';
 import Breadcrumbs from '../../../components/Breadcrumbs';
 
-const styles = () => ({
-  parameters: {
-    float: 'left',
-    marginTop: -10,
-  },
+const ParametersHeader = styled('div')({
+  marginTop: -10,
+});
+
+const StyledSearchInput = styled(SearchInput)({
+  float: 'left',
+});
+
+const CreateSectorButton = styled('div')({
+  float: 'right',
 });
```

Comment:
We chose to favor `style` props in the jsx instead of using Material `styled` utility.